docs/getting-started/summaryofchanges.md

```diff
@@ -15,13 +15,20 @@ Welcome to Zowe Active Development documentation.
 
 **New compatible plug-ins:**
 
-In addition to the CICS and Db2 Zowe CLI plug-ins, the following plug-ins are available for Active Development:
+The following plug-ins are compatible *only* with Zowe CLI Active Development version:
 
-- The [IBM IMS Plug-in for Zowe CLI](../user-guide/cli-imsplugin.md).
-- The [IBM MQ Plug-in for Zowe CLI](../user-guide/cli-mqplugin.md).
+- [IBM® z/OS FTP Plug-in for Zowe CLI](../user-guide/cli-mqplugin.md).
+- [IBM® IMS™ Plug-in for Zowe CLI](../user-guide/cli-imsplugin.md).
+- [IBM® MQ Plug-in for Zowe CLI](../user-guide/cli-mqplugin.md).
+- [Secure Credential Store for Zowe CLI](../user-guide/cli-mqplugin.md).
+
+<!-- TODO Add SCS and FTP plugin link paths to list   -->
+
+**Note:** This is not a comprehensive list of plug-ins. For a complete list of plug-ins compatible with the Active Development version, see [Extending Zowe CLI - Active Development](https://docs.zowe.org/active-development/user-guide/cli-extending.html). For a complete list of plug-ins compatible with the `@lts-incremental` version, see [Extending Zowe CLI - Stable](https://docs.zowe.org/stable/user-guide/cli-extending.html).
 
 **Features and enhancements:**
 
+- The `zowe files copy` command was added for copying the contents of a data set or member to another data set or member. [#580](https://github.com/zowe/zowe-cli/pull/580)
```

Comment:
Was this the only human change in this PR?
If so, I think we need to do something with the `.vuepress`  folder.

Reply:
Good morning @zFernand0  - I added the new web help files (.zip, HTML, and PDF). This causes many files to be changed. Can you clarify what the issue is? I plan to build and test the web help links shortly, then also add the LTS web help in another PR.

Reply:
I was just wondering if most of these files are in fact generated by a script, it might be worth sourcing them on a separate repo (or somewhere else) so that it doesn't affect the PR review process. i.e. do something so that people can see what you want them to see 😄 

@jackjia-ibm, any suggestions?

Reply:
Would make sense if it's possible - and it's probably unnecessary to have the doc-site broken link tests scan these files. 

@zFernand0 I can also show you the steps I take to add these files, as I'm sure that updating them could be automated. But that's another story.